source/fundamentals/gridfs.txt

```diff
@@ -0,0 +1,707 @@
+.. _csharp-gridfs:
+
+=================
+Store Large Files
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: binary large object, blob, storage, code example
+
+Overview
+--------
+
+In this guide, you can learn how to store and retrieve large files in
+MongoDB by using **GridFS**. GridFS is a specification implemented by 
+the {+driver-short+} that describes how to split files into chunks when storing them
```

Comment:
```suggestion
the {+driver-short+} that allows you to split files into chunks when storing them
```

Reply:
this is one of questions about technical accuracy vs. reader understanding, because it is correct that GridFS is an implemented specification, and specifications can only describe--they don't do anything themselves. BUT I get that that may be more detail than a user needs. Reworded.